docs/spec/transactions/transaction.json

```diff
@@ -50,8 +50,7 @@
             "regexProperties": true,
             "patternProperties": {
                 "^[^.*\"]*$": {
-                    "$ref": "mark.json",
-                    "maxLength": 1024
```

Comment:
This length restriction did not make sense, as it doesn't refer to the properties key but the value and the value is a `mark`.